### DISCUSSION TITLE
Is it possible to auto-update all the fields of a note when adding/removing a Class field?

### DISCUSSION THREAD

**Discussion by gitroms** [mdelobelle/metadatamenu#283] [Q&A]
Hi,
The question is in the title 😀
Imagine you have lots of notes with a defined template that uses a specific Class, and you need to update the Class with a new Field.
Do we have to update manually each note or is there an option to update all the Notes at once?
Thank you

**Comment by msageryd** [comment]
There is a discussion on this. Please chime in and add your thumbs up =)

#591
